src/django_elasticsearch_dsl_drf/viewsets.py

```diff
@@ -17,7 +17,7 @@
 
 from .pagination import PageNumberPagination
 from .utils import DictionaryProxy
-
+import re
```

Comment:
This import becomes irrelevant and shall be removed.

Reply:
silly me !! will clean that